package.json

```diff
@@ -11,7 +11,7 @@
     "stop": "nps --prefix=stop",
     "restart": "nps --prefix=restart",
     "delete": "nps --prefix=delete",
-    "rebuild-packages": "yarn workspaces foreach run build",
+    "build": "yarn workspace fxa-shared run build && yarn workspace fxa-auth-client run build && yarn workspace fxa-graphql-api run build",
```

Comment:
Does this run on `yarn start`? or would we have to call `yarn build` now as part of starting up fxa before calling `yarn start`? Ideally this fix could be rolled into the existing `yarn start`.

One thought: Instead of hardcoding the ones we know about now, is it possible to check each package's `package.json` to see if any of the fxa packages depend on any other fxa packages (e.g. in `dependencies`), and create a `Set` from that to execute `yarn build` commands on?

Just trying to think about how we can ensure this works even if the packages/dependencies change.

Maybe that's overkill though. 😅